homeassistant/components/ps4/media_player.py

```diff
@@ -201,16 +208,19 @@ def get_title_data(self, title_id, name):
         app_name = None
         art = None
         try:
-            app_name, art = self._ps4.get_ps_store_data(
+            title = self._ps4.get_ps_store_data(
                 name, title_id, self._region)
-        except TypeError:
+            app_name = title.name
+            art = title.cover_art
+        except (AttributeError, TypeError):
```

Comment:
Why would there be an attribute error? Attributes should always be set when the instance is instantiated.

Reply:
When does the type error happen?

It would be better if the library raised its own exceptions, and we handle expected return values with checks instead.

Reply:
Corrected in library defined all attributes in object init. These errors occur when an attribute does not exist in the return of an http request to fetch the cover art.